irohad/consensus/yac/storage/yac_block_storage.hpp

```diff
@@ -105,18 +104,18 @@ namespace iroha {
         bool uniqueVote(VoteMessage &vote);
 
         /**
-         * Verify that vote has same proposal and
-         * blocks hashes with storage
+         * Verify that vote has the same round attached as the storage
```

Comment:
It compares the whole hash, not only round. Please reflect this in the docstring.